src/app/routes/cpsAsset/utils/getAdditionalPageData.js

```diff
@@ -0,0 +1,45 @@
+import isEmpty from 'ramda/src/isEmpty';
+import { STORY_PAGE } from '#app/routes/utils/pageTypes';
+import getAssetType from './getAssetType';
+import fetchPageData from '../../utils/fetchPageData';
+
+const pageTypeUrls = (assetType, service) => {
+  switch (assetType) {
+    case STORY_PAGE:
+      return [
+        {
+          name: 'mostRead',
+          path: `/${service}/mostread`,
```

Comment:
Just a note for this path. I have created this Utils: https://github.com/bbc/simorgh/blob/52cbca753e7ca408fe169ea5ce970997f58b0e22/src/app/lib/utilities/getMostReadUrls/index.js#L1-L5 for variant path

Reply:
ohhh thats nice, did not know. I will update this PR to use that util. Thanks @thekp